### ISSUE TITLE
4 and 5 lead Spice transistors seem to have issues

### ISSUE THREAD

**Issue by tomhajjar** [ra3xdh/qucs_s#391]
The "new" 4 and 5 lead Spice transistors seem to have issues. 

I'm having issues getting a HICUM project to work using "model cards" for a 5 lead NPN transistor.

The "Edit Component Properties" entries and/or the description are wrong and the generated netlist is wrong.

For comparison, an old Qucs HICUM Verilog project was used to see what parameters are should be in the 5 lead NPN transistor.

![2023-12-01_184658](https://github.com/ra3xdh/qucs_s/assets/102838604/a3a79ab4-f560-470b-8912-b5fba087d0eb)

![2023-12-01_185527](https://github.com/ra3xdh/qucs_s/assets/102838604/aba10dc3-3451-4879-bebf-ffe5a07dd931)

![2023-12-01_185743](https://github.com/ra3xdh/qucs_s/assets/102838604/1706a68c-050a-4d3a-9e92-979f98b35df0)

![2023-12-01_165225](https://github.com/ra3xdh/qucs_s/assets/102838604/9762d872-9ccf-420e-bdae-6cb69beba373)

[HICUM2_prj.zip](https://github.com/ra3xdh/qucs_s/files/13532759/HICUM2_prj.zip)

**Comment by ra3xdh** [comment]
> the generated netlist is wrong.

Yes, there is a bug with spice entry. Parameters are shifted. I will provide a fix soon. 

> an old Qucs HICUM Verilog project was used to see what parameters are should be in the 5 lead NPN

The new device should not follow old Qucs HICUM specification. The Qucs device was created for Qucsator+ADMS. The HICUM was removed from Qucsator some time ago. And I removed this device from GUI too. The new device uses another concept. It should contain only reference to the .MODEL, but not list of properties. This will allow to attach this device to PDK libraries that are distributed as a set of subcircuits.
